.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-version: ['7.4', '8.0', '8.1']
```

Comment:
All the changes to ci.yml are regressions.

Reply:
I could've sworn I've chosen "ours" instead of "theirs" 😞

Do we care about a clean history with regards to such conflict resolution mistakes, eg should I fix on top, or should I rather rewrite the history?

Reply:
Do whatever is convenient for you :)

Reply:
Well, first rewrite attempt completely bricked the branch, but second one worked :)